lib-multisrc/colamanga/src/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt

```diff
@@ -166,7 +171,14 @@ abstract class ColaManga(
         }
     }
 
-    override fun chapterListSelector(): String = "div:not(.fed-hidden) > div.all_data_list > ul.fed-part-rows a"
+    override fun chapterListSelector() = throw UnsupportedOperationException()
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val document = response.asJsoup()
+        return document.select("div:not(.fed-hidden) > div.all_data_list > ul.fed-part-rows a").map{ chapterFromElement(it) }.apply {
+            this[0].date_upload = dateFormat.parse(document.selectFirst("span.fed-text-muted:contains($lastUpdated) + a")?.text()).getTime()
```

Comment:
```suggestion
            this[0].date_upload = dateFormat.tryParse(document.selectFirst("span.fed-text-muted:contains($lastUpdated) + a")?.text())
```
import `keiyoushi.utils.tryParse`